hyperon_das_atomdb/adapters/server_db.py

```diff
@@ -0,0 +1,277 @@
+import json
+from typing import Any, Dict, List, Optional, Tuple
+
+from requests import Session, exceptions
+
+from hyperon_das_atomdb.exceptions import (
+    ConnectionServerException,
+    NodeDoesNotExistException,
+)
+from hyperon_das_atomdb.i_database import IAtomDB
+from hyperon_das_atomdb.utils.decorators import retry
+
+
+class ServerDB(IAtomDB):
+    """A concrete implementation using servers databases.
+    AwsLambda and OpenFaas"""
+
+    def __repr__(self) -> str:
+        return "<Atom database Server>"  # pragma no cover
+
+    def __init__(
+        self,
+        database_name: str = 'das',
```

Comment:
This should not be a parameter here. Users are not aware of DB names. Actually users don´t even know which DB engines are being used.

Reply:
ok